inst/tests/tests.Rraw

```diff
@@ -17596,3 +17596,7 @@ DT = data.table(a=1L)
 test(2186, DT[, if (TRUE) .(a=1L) else .(a=1L, b=2L)], DT,
      warning='j may not evaluate to the same number of columns for each group')
 
+# col.names='none' should apply when wrapping too, #4270
+DT = setDT(replicate(getOption('width'), 1, simplify = FALSE))
+test(2187, {print(DT, col.names='none'); TRUE}, notOutput="v")
```

Comment:
I checked this test fails correctly in dev before this PR. Good.
Even though `notOutput=` is case insensitive and the test does work, I feel that capital `"V"` would leave a little less scope for doubt since the column names in this test are lots of default `V<num>`. So I'll change that `"v"` to be `"V"`